src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -94,7 +95,12 @@ class MessagingWidget extends React.Component {
       <div id="msg-widget">
         <h2>Check Secure Messages</h2>
         {content}
-        <p><Link href="/health-care/messaging" onClick={recordDashboardClick('view-all-messages')}>View all your secure messages</Link>.</p>
+        <p>
+          {isBrandConsolidationEnabled() ?
+            (<a href="https://www.myhealth.va.gov/mhv-portal-web/secure-messaging" target="_blank">View all your secure messages</a>) :
+            (<span><Link href="/health-care/messaging" onClick={recordDashboardClick('view-all-messages')}>View all your secure messages</Link>.</span>)
```

Comment:
just curious why this `Link` needs to be wrapped in a `span` now.

Reply:
There's a period at the end. It's gross but the ternary can only return a single node.